services/index/src/helpers.js

```diff
@@ -169,6 +169,35 @@ export const taskUtils = {
     // Fetch results
     return fetchResults(query ? query.continuationToken : {});
   },
+
+  async findTasksAtIndexes(db, { indexes }, { query } = {}) {
+    assert(_.isArray(indexes), 'indexes must be an Array');
```

Comment:
`input` schema validation would make sure only valid payloads would be passed, but it doesn't hurt making it future-proof also 👍

Reply:
sure :)